test/main.ml

```diff
@@ -0,0 +1,32 @@
+(** testing*)
+open OUnit2
+open GenerateLetters
+
+(** [char_to_string_list] is a string list representation of a char list*)
+let rec char_to_string_list = function 
+  | [] -> []
+  | h :: t -> (Char.escaped h) :: (char_to_string_list t)
+
+let pp_string s = "\"" ^ s ^ "\""  
+
+(** [pp_list pp_elt lst] pretty-prints list [lst], using [pp_elt] to
+    pretty-print each element of [lst]. TAKEN FROM A2*)
+let pp_list pp_elt lst =
+  let pp_elts lst =
+    let rec loop n acc = function
+      | [] -> acc
+      | [ h ] -> acc ^ pp_elt h
+      | h1 :: (_h2 :: _t as t') ->
```

Comment:
what does this `as` statement do?

Reply:
^ also confused why it's necessary to bind t', do we need it later

Reply:
i was just using these as ways to print what start_game did sorry